src/transformers/__init__.py

```diff
@@ -255,6 +255,7 @@
     "models.mobilebert": ["MOBILEBERT_PRETRAINED_CONFIG_ARCHIVE_MAP", "MobileBertConfig", "MobileBertTokenizer"],
     "models.mpnet": ["MPNET_PRETRAINED_CONFIG_ARCHIVE_MAP", "MPNetConfig", "MPNetTokenizer"],
     "models.mt5": ["MT5Config"],
+    "models.mvp": ["MvpConfig", "MvpTokenizer"],
```

Comment:
(nit) would prefer `MVPConfig` and `MVPTokenizer` as names so that they align with other models, but not too important to me either

Reply:
According to the response of @sgugger , maybe it should be `MVP`.